src/Sulu/Bundle/AdminBundle/Resources/js/components/ColorPicker/tests/ColorPicker.test.js

```diff
@@ -1,64 +1,79 @@
 // @flow
 import React from 'react';
-import {mount} from 'enzyme';
+import {fireEvent, render, screen} from '@testing-library/react';
```

Comment:
would be nice to use the `@testing-library/user-event` package instead of `fireEvent` 🙂 see https://github.com/sulu/sulu/pull/6783 for some examples